en/console-and-shells/plugin-shell.rst

```diff
@@ -33,6 +33,15 @@ Adding the ``-b`` or ``-r`` switch to the load task will enable loading of the p
     // Load the routes.php from the plugin
     Plugin::load('MyPlugin', ['routes' => true]);
 
+If you are loading a plugin that only provides CLI tools (like bake, or
+migrations) you can update your ``bootstrap_cli.php`` with::
```

Comment:
About the migrations plugin, that is partially true. While most of the operations are done using the CLI tools, there's a class that allows migrating in a non-shell environment.
But that's nitpicking.